### ISSUE TITLE
2.4: Rename Passage titles by clicking on them

### ISSUE THREAD

**Issue by hituro** [klembot/twinejs#1013]
### Is your feature request related to a problem? Please describe.

Currently clicking, right-clicking, or double-clicking the title of an open passage collapses it. This is counter-intuitive in terms of how windows are renamed in many other UIs, where double or right clicking a title allows a rename.

### Describe the solution you'd like.

Double or right clicking a passage title should allow you to rename it.

### Describe alternatives you've considered.

It would be reasonable for a single click on the passage title to allow a rename, with the collapse tied to double-click or a click on the collapse arrow only.

### Presubmission checklist

- [ ] I am interested in working on code that would implement this feature request. (This is not required to submit a suggestion.)
- [X] I have done a search and believe that an issue does not already exist for this idea in the GitHub repository.
- [X] I have read and agree to abide by this project's Code of Conduct.

**Comment by hituro** [comment]
Addendum to this. After some days of using 2.4 in production, I have accidentally collapsed passage windows while attempting to rename them a number of times, and find passage renaming in general more onerous than it was in 2.3

**Comment by hhrichards** [comment]
I agree with this. I've also clicked the passage name many times, expecting it to be editable and collapsed it! Clicking the passage title should make it editable imo. I can't see any need to collapse a passage dialog.

**Comment by drball** [comment]
I think this would be much more intuitive and simpler than having a separate button called "Rename".
